msibi/state.py

```diff
@@ -33,9 +51,9 @@ class State(object):
         True if each query trajectory is backed up (default=False)
 
     """
-    def __init__(self, k, T, state_dir='', traj_file=None, top_file=None,
+    def __init__(self, kT, state_dir='', traj_file=None, top_file=None,
                  name=None, backup_trajectory=False):
-        self.kT = k * T
+        self.kT = kT
```

Comment:
In HOOMD 1.0, the value you pass to the integrator is still kT, but the parameter you pass is called T. They changed it to kT because everybody new to HOOMD seemed to have issues with that, so this is still good for HOOMD. We'll have to make this clear in the docs that for other engines you make have to pass kT, e.g., ```state = State(kT=305*8.314e-3, ...)```